### PR TITLE
Omit Result from Activation if Empty

### PR DESCRIPTION
**Pull request by dubeejw**
- Do not display the result field from an activation record if it empty

Closes https://github.com/openwhisk/openwhisk/issues/1467

### CODE REVIEW

File: tools/cli/go-whisk/whisk/activation.go [hunk 1]
```diff
@@ -44,10 +44,10 @@ type Activation struct {
...
 
 type Response struct {
-    Status     string `json:"status"`
-    StatusCode int    `json:"statusCode"`
-    Success    bool   `json:"success"`
-    Result     `json:"result"`
+    Status     string   `json:"status"`
+    StatusCode int      `json:"statusCode"`
+    Success    bool     `json:"success"`
+    Result     Result  `json:"result,omitempty"`
```
Comment (rabbah) [inline tools/cli/go-whisk/whisk/activation.go]: will this omit `{}`?

**Comment by markusthoemmes** [comment]
Why delete all tests? Bad push?

**Comment by dubeejw** [comment]
PG1 801.

**Comment by dubeejw** [comment]
PG2 542

**Comment by dubeejw** [comment]
PG2 552

**Comment by dubeejw** [comment]
@rabbah, any comments on this one?

**Comment by dubeejw** [comment]
PG1 885

**Pull request merged**